setup.py

```diff
@@ -27,13 +27,13 @@ def get_requirements(filename='requirements.txt'):
 
 setup(
     name='fmn.sse',
-    version='0.1.1',
+    version='0.2.0',
     description='Real time feed for fedmsg',
     long_description=get_description(),
-    author='Szymon M',
-    author_email='skrzepto@gmail.com',
+    author='Fedora Infrastructure Team and Szymon M',
```

Comment:
Yeah. I don't have strong feelings one way or the other, so I'm inclined to leave it as is. I think I'd prefer to maintain an AUTHORS file and include that in the packaging.